@kiva/kv-shop/src/components/KvPaymentSelect.vue

```diff
@@ -89,7 +92,7 @@ export default defineComponent({
 			paymentTypes,
 			preselectVaultedPaymentMethod,
 		} = toRefs(props);
-		const container = ref<HTMLInputElement | null>(null);
+		const container = ref<any>(null);
```

Comment:
Why didn't `HTMLInputElement` work here?

Reply:
Typescript was complaining about this type not matching the dropin option type, with a really long error message that I wasn't able to fully read. At first I thought that the mismatch was due to the options type being `HTMLElement | string`, but changing the container type to HTMLElement didn't change the reported issue, which makes sense as HTMLInputElement probably implements HTMLElement.

It seems that the HTMLElement type we have doesn't match the one that the dropin has. I couldn't easily find where either type was coming from, so I changed this to `any` to avoid the typescript error.

I think I could use `typeof DropinOptions.container` here instead though